src/components/Home/Hero/HeroSection.tsx

```diff
@@ -0,0 +1,191 @@
+      className
+    )}
+    {...props}
+  >
+    {children}
+  </button>
+)
+
+const HeroSection = () => {
+  const stars = useStars()
+
+  return (
+    <div className="md:border-y-2 md:border-solid md:border-y-light">
+      {/* Title Section */}
+      <SectionWrapper>
+        <Section>
+          <img src="/img/logos/dvcx.svg" alt="DVCx Logo" className="h-28" />
+          <div
+            className={cn(
+              'flex flex-col gap-4 flex-1 justify-between',
+              'lg:flex-row lg:items-center'
+            )}
+          >
+            <h2 className="text-2xl md:text-3xl font-medium">
+              Dataset factory
+            </h2>
+            <Badge className="bg-dark text-light">Coming soon</Badge>
+          </div>
+        </Section>
+        <Section>
+          <img src="/img/logos/dvc.svg" alt="DVC Logo" className="h-28" />
+          <div
+            className={cn('flex flex-col gap-4', 'lg:flex-row lg:items-center')}
+          >
+            <h2 className="text-2xl md:text-3xl font-medium">
+              Data and model versioning
+            </h2>
+            <Link
+              href="https://github.com/iterative/dvc"
+              className="no-underline hover:opacity-80"
+            >
+              <Badge className="bg-dark text-light">
+                {stars && shortenNumber(stars, 1)}
+                <StaticImage
+                  src="../../../../static/img/landing/star-github.svg"
+                  alt="Star Github"
+                />
+              </Badge>
+            </Link>
+          </div>
+        </Section>
+      </SectionWrapper>
+      {/* Visualization Section */}
+      <SectionWrapper maxWidth="2xl">
+        <div className={cn(styles.gridContainer)}>
+          <StaticImage
+            src={'../../../../static/img/landing/Hero Visualization.svg'}
+            quality={100}
+            formats={['avif', 'webp', 'auto']}
+            className={styles.heroViz}
+            alt="Visualization"
+          />
+          <div className={cn(styles.heroDesc, styles.dvcxDesc)}>
+            <strong>Explore</strong> and <strong>enrich</strong> annotated
+            datasets with custom embeddings, auto-labeling, and bias removal at
+            billion-file scale — without modifying your data.
+          </div>
+
+          <div className={cn(styles.heroDesc, styles.dvcDesc)}>
+            <strong>Connect</strong> to versioned data sources and code with
+            pipelines, <strong>track</strong> experiments,{' '}
+            <strong>register</strong> models — all based on GitOps principles.
+          </div>
+        </div>
+      </SectionWrapper>
+
+      {/* CTA Section */}
+
+      <SectionWrapper>
+        <Section>
+          <CTAButton
+            className="bg-dark text-light"
+            onClick={() => {
+              navigate('#get-started-dvcx', { state: { focusInput: true } })
+            }}
+          >
+            <span>
+              Download DVC<sup>X</sup>
```

Comment:
I would not label buttons that navigate/scroll "Download".
Maybe "Get DVC/DVCx" ?